python/ray/ml/preprocessor.py

```diff
@@ -29,31 +25,63 @@ class Preprocessor(abc.ABC):
     fitting, and uses these attributes to implement its normalization transform.
     """
 
+    class FitStatus(str, Enum):
+        """The fit status of preprocessor."""
+
+        NOT_FITTABLE = "NOT_FITTABLE"
+        NOT_FITTED = "NOT_FITTED"
+        # Only meaningful for Chain preprocessors
+        PARTIALLY_FITTED = "PARTIAL_FITTED"
```

Comment:
Name these the same?

```suggestion
        PARTIALLY_FITTED = "PARTIALLY_FITTED"
```